packages/salesforce-adapter/src/constants.ts

```diff
@@ -76,6 +76,19 @@ export enum FIELD_TYPE_NAMES {
   FILE = 'File',
 }
 
+// Flow element type name
+export const FLOW_NODE = 'FlowNode'
+
+// FlowStart field name
+export const TARGET_REFERENCE = 'targetReference'
+
+// FlowNode field names
+export enum FLOW_NODE_FIELD_NAMES {
+  NAME = 'name',
+  LOCATION_X = 'locationX',
+  LOCATION_Y = 'locationY',
+}
+
```

Comment:
```suggestion
// Flow constants
export const FLOW_NODE = 'FlowNode'
export const TARGET_REFERENCE = 'targetReference'
export enum FlowNodeField {
  Name = 'name',
  LocationX = 'locationX',
  LocationY = 'locationY',
}
```
1. I think we should follow `PascalCase` for enums. It's more readable IMO.
2. These comments are confusing to me. I'd simply put them all under section of `Flow constants`

Reply:
I fixed the comments.
Regarding the enum, all the enums in the module use upper snake case, Isn't it best to maintain consistency?

Reply:
Some of them do some don't, you can keep it as UPPER_SNAKE_CASE.